lib/roken/getaddrinfo-test.c

```diff
@@ -129,7 +129,8 @@ main(int argc, char **argv)
 	return 0;
     }
 
-    rk_SOCK_INIT();
+    if (rk_SOCK_INIT())
```

Comment:
This will end up being

   if (0) errx(1, ....);

which will generate a warning of its own (at least in Coverity).  Most likely some variation of a dead code warning.   

My opinion is that if this change will be made that rk_SOCK_INIT() should become a proper function and not just a macro.

Reply:
It is now a proper inline function.  Long term we can't really win against Coverity and still do the cross-platform case where it is a call that could fail on windows, but this may be a better compromise and makes a real-world improvement in the number of warnings we need to allow (where other cases might be real bugs)